src/test/java/org/broadinstitute/consent/http/db/ElectionDAOTest.java

```diff
@@ -240,6 +240,12 @@ public void testFindAllDacsForElectionIds() {
     assertEquals(dac.getDacId(), dacRecord.getDacId());
   }
 
+  @Test
+  public void testFindAllDacsForElectionIds_NotFound() {
```

Comment:
Nit: Instead of using `NotFound`, use something like `EmptyList`. `NotFound` has exception connotations (due to `NotFoundException` in Java) which can cause misinterpretations on a quick sight read.